docs/use-existing-kafka.md

```diff
@@ -171,3 +171,38 @@ When using SSL to secure kafka communication, you must set `KAFKA_ENABLE_TLS`, a
 * `KAFKA_CACERTS` to check server certificate
 * `KAFKA_CERT` and `KAFKA_KEY` to check client certificate
 * `KAFKA_INSECURE` to skip TLS verfication
+```yaml
+$ echo '
```

Comment:
I think you can remove the `$echo '` part (I know it's like that in the other example but it's not clarifying anything